instaloader/structures.py

```diff
@@ -602,6 +602,15 @@ def video_view_count(self) -> Optional[int]:
             return self._field('video_view_count')
         return None
 
+    @property
+    def video_play_count(self) -> Optional[int]:
+        """Play count of the video, or None.
+
+        .. versionadded:: 4.2.6"""
```

Comment:
```suggestion
        .. versionadded:: 4.14.3"""
```

Then we can merge this pull request, it looks fine to me.